### ISSUE TITLE
Chart postgresql-ha fails if commonAnnotations not set and service.annotations is set

### ISSUE THREAD

**Issue by wolf31o2** [bitnami/charts#6064]
**Which chart**:
postgresql-ha 6.8.2

**Describe the bug**
Set `service.annotations` without setting `commonAnnotations` and the service will fail with a template error.

**To Reproduce**
Create a YAML file:
```yaml
service:
  annotations:
    foo: bar
```
Run it via helm template:
```
helm template test bitnami/postgresql-ha -f test.yaml
```

**Expected behavior**
Pass while setting the correct annotations on the Service resource for pgpool.

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.2.0", GitCommit:"e11b7ce3b12db2941e90399e874513fbd24bcb71", GitTreeState:"clean", GoVersion:"go1.13.10"}
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.2", GitCommit:"faecb196815e248d3ecfb03c680a4507229c2a56", GitTreeState:"clean", BuildDate:"2021-01-13T13:28:09Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"17+", GitVersion:"v1.17.12-eks-7684af", GitCommit:"7684af4ac41370dd109ac13817023cb8063e3d45", GitTreeState:"clean", BuildDate:"2020-10-20T22:57:40Z", GoVersion:"go1.13.15", Compiler:"gc", Platform:"linux/amd64"}
```

**Additional context**

**Comment by alemorcuq** [comment]
Thanks for reporting this, @wolf31o2. I have created a PR addressing the issue.

**Comment by alemorcuq** [comment]
Hi, @wolf31o2! The fix will be released in version 6.8.4.
